experiment/resultstore/convert.go

```diff
@@ -124,6 +127,32 @@ func convert(project, details string, url gcs.Path, result downloadResult) (resu
 				URL:         buildLog, // ensure build-log.txt appears as the invocation log
 			},
 		},
+		Properties: []resultstore.Property{
+			{
+				Key:   "Job Name",
+				Value: jobName,
+			},
+			{
+				Key:   "Started Timestamp",
```

Comment:
This exists elsewhere: https://github.com/googleapis/googleapis/blob/master/google/devtools/resultstore/v2/common.proto#L49

Reply:
I wish they had used ISO date format in the resultstore UI. Even though it is start_time in the proto, the result store named it Run Date. But fair, I removed this entry.